backend/api/database.py

```diff
@@ -17,3 +17,40 @@ def get_user_by_email(db: Session, email: str):
     if len(user) > 1:
         raise MultipleResultsFound
     return user[0]
+
+
+def get_all_users(db: Session):
+    users = db.query(UserAccount).all()
+    if not users:
+        raise NoResultFound
```

Comment:
:thinking: -- Do we really want to raise an exception if the result is an empty array?

Reply:
I guess we wouldn't. In this particular instance, it's not a bad thing if we try to get all the users in the db and none currently exist.